asset_feed.tf

```diff
@@ -0,0 +1,62 @@
+# Subscribe to the cloud asset inventory
+
+# The topic where the resource change notifications will be sent.
+resource "google_pubsub_topic" "feed_output" {
+  name = "${local.prefix}asset-feed"
+}
+
+# Project needs to have enabled cloud asset inventory API
+
+resource "google_cloud_asset_organization_feed" "organization_feed" {
+  count           = var.organization_id == "" ? 0 : 1
+  billing_project = var.project_id
+  org_id          = var.organization_id
+  feed_id         = "${local.prefix}stacklet-resource-feed"
```

Comment:
Should we have a separator between the prefix and the rest of the string here?   So instead of `infobloxstacklet-resource-feed` it comes out as `infoblox-stacklet-resource-feed`?